<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(rgb(133, 174, 207));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="global_top">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ТОП-50 світу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="my_songs_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Мої пісні</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>-50</y>
      <width>20</width>
      <height>831</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="recommended_songs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Рекомендації</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>811</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
     </font>
    </property>
    <property name="locale">
     <locale language="Ukrainian" country="Ukraine"/>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>-10</y>
      <width>811</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Насолоджуйтесь гармонійним поєднанням ритму і тексту та знайдіть свій власний саундтрек.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="add_song">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Додати пісню</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_song">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Видалити пісню</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_line">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>10</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пошук</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ukraine_top">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ТОП-50 України</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMusic_recommendations">
    <property name="title">
     <string>Music recommendations</string>
    </property>
   </widget>
   <addaction name="menuMusic_recommendations"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
